### PR TITLE
APIDB-9753 add missing capabilities

### PR DESCRIPTION
**Pull request by francesco-fipertani-vonage**
# Description

# Checklist

- [ ] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/application.v2.yml [hunk 1]
```diff
@@ -699,131 +375,229 @@ components:
...
+      properties:
+        webhooks:
           type: object
-          description: "Application privacy config"
           properties:
-            improve_ai:
-              type: boolean
-              description: "If set to `true`, Vonage may store and use your content and data for the improvement of Vonage's AI based services and technologies."
-              example: true
+            room_changed:
```
Comment (pardel) [inline definitions/application.v2.yml:555]: Needs a description.

File: definitions/application.v2.yml [hunk 2]
```diff
@@ -699,131 +375,229 @@ components:
...
+              type: object
+              properties:
+                address:
+                  type: string
+                  example: https://example.com/webhooks/event
+                http_method:
+                  type: string
+                  example: POST
+            session_changed:
```
Comment (pardel) [inline definitions/application.v2.yml:564]: Needs a description.

File: definitions/application.v2.yml [hunk 3]
```diff
@@ -699,131 +375,229 @@ components:
...
+              type: object
+              properties:
+                address:
+                  type: string
+                  example: https://example.com/webhooks/event
+                http_method:
+                  type: string
+                  example: POST
+            recording_changed:
```
Comment (pardel) [inline definitions/application.v2.yml:573]: Needs a description.

File: definitions/application.v2.yml [hunk 4]
```diff
@@ -699,131 +375,229 @@ components:
...
+              type: object
+              properties:
+                address:
+                  type: string
+                  example: https://example.com/webhooks/event
+                http_method:
```
Comment (pardel) [inline definitions/application.v2.yml:579]: shouldn't this be an enum?

File: definitions/application.v2.yml [hunk 5]
```diff
@@ -699,131 +375,229 @@ components:
...
+                  type: string
+                  example: POST
+            session_changed:
+              type: object
+              properties:
+                address:
+                  type: string
+                  example: https://example.com/webhooks/event
+                http_method:
```
Comment (pardel) [inline definitions/application.v2.yml:570]: shouldn't this be an enum?

File: definitions/application.v2.yml [hunk 6]
```diff
@@ -699,131 +375,229 @@ components:
...
-              type: boolean
-              description: "If set to `true`, Vonage may store and use your content and data for the improvement of Vonage's AI based services and technologies."
-              example: true
+            room_changed:
+              type: object
+              properties:
+                address:
+                  type: string
+                  example: https://example.com/webhooks/event
+                http_method:
```
Comment (pardel) [inline definitions/application.v2.yml:561]: shouldn't this be an enum?

**Review by pardel — CHANGES_REQUESTED** [review]
This is a regression on the previous specs - can we make sure we're iterate on the current version (https://developer.vonage.com/api/application.v2). Lots of details are missing.

File: definitions/application.v2.yml [hunk 7]
```diff
@@ -699,131 +375,229 @@ components:
...
+                  type: string
+                  example: POST
+
+    VerifyCapability:
+      type: object
+      description: Verify related configuration
```
[the remark this answers is not in the document]
Comment (francesco-fipertani-vonage) [reply definitions/application.v2.yml:625]: Not sure what you mean with this. This is the v2 documentation...

File: definitions/application.v2.yml [hunk 8]
```diff
@@ -699,131 +375,291 @@ components:
...
+                  example: POST
+                  enum:
+                    - GET
+                    - POST
+
+    VoiceCapability:
+      type: object
+      description: Voice related configuration
+      properties:
+        payment_enabled:
```
Comment (pardel) [inline definitions/application.v2.yml:477]: Is this property public? It's an addition on this spec.
If public, we need a better description.

File: definitions/application.v2.yml [hunk 9]
```diff
@@ -699,131 +375,291 @@ components:
...
+          type: boolean
+        signed_callbacks:
```
Comment (pardel) [inline definitions/application.v2.yml:479]: Is this property public? It's an addition on this spec.
If public, we need a better description.

File: definitions/application.v2.yml [hunk 10]
```diff
@@ -699,131 +375,291 @@ components:
...
+          type: boolean
+        conversation_ttl:
```
Comment (pardel) [inline definitions/application.v2.yml:481]: Is this property public? It's an addition on this spec.
If public, we need a better description.

File: definitions/application.v2.yml [hunk 11]
```diff
@@ -699,131 +375,291 @@ components:
...
+          type: integer
+        payments:
```
Comment (pardel) [inline definitions/application.v2.yml:483]: Is this property public? It's an addition on this spec.
If public, we need a better description (including for its properties).

**Review by pardel — CHANGES_REQUESTED** [review]

File: definitions/application.v2.yml (hunk 8 above)
Comment (francesco-fipertani-vonage) [reply definitions/application.v2.yml]: Checked with Jamie and he didn't want that flag in the docs

**Review by pardel — APPROVED** [review]

**Pull request merged**